codebase/superdarn/src.bin/tk/plot/time_plot.1.8/time_plot.c

```diff
@@ -1419,7 +1422,8 @@ int main(int argc,char *argv[]) {
                        tplot.rxrise,300,&rho,&tlat,&lon,chisham);
 
           if (tlat<blat) {
-            tmp=blat;
+            // TODO: this isn't actually swapping things
+            // tmp_swap=blat;
```

Comment:
Shouldn't this be something like

```
tmp_swap=blat;
blat=tlat;
tlat=tmp_swap;
```

to ensure the swap?

Reply:
Yea, but wasn't sure if a swap was actually meant.

Reply:
Since the purpose of this PR is to remove warnings, perhaps we can look into this separately to determine the correct/intended behaviour? It looks like @mts299 noticed this while working on the warnings but it's actually a separate issue.

Reply:
Made an issue for it.